ansible/filter_plugins

```diff
@@ -0,0 +1 @@
+../../kayobe/ansible/filter_plugins/
```

Comment:
Is a relative path reference the best option here?  I was wondering if a playbook dir variable might exist we could use.

Reply:
This is a symlink, so I don't think we can use a variable. The path is relative to the directory structure set up by https://github.com/markgoddard/beokay.